src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
     background: linear-gradient(90deg, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.0));
 }
 
+.epigenome-body-map-container {
+    border: 2px solid $Homo-sapiens;
+    position: relative;
+    .body-list-top {
+        border-top: 0;
+    }
+    &.expanded {
+        max-width: 675px;
+        position: absolute;
+        z-index: 99999;
+        background: white;
+        left: 0;
+        right: 0;
+        margin: auto;
+        top: 120px;
+        @media screen and (max-width: $screen-sm-min) {
+            width: 95%;
+        }
+    }
+    &.collapsed {
+        width: 0;
+        height: 0;
+        border: 0;
+        overflow: hidden;
+    }
+    .collapse-body-map {
+        cursor: pointer;
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        font-size: 1rem;
+        border: 0;
+    }
+    .body-list {
+        .body-list-inner {
+            text-align: center;
+        }
+    }
+    .body-list-inner {
+        li {
+            display: inline;
+            &:after {
+                content: " ";
+                word-spacing: 0em;
+                letter-spacing: 1em;
+                background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAOwAAADsABataJCQAAABl0RVh0U29mdHdhcmUAcGFpbnQubmV0IDQuMC4xMkMEa+wAAAAnSURBVBhXY/Dz89MA4sNA/B9Ka4AEYQIwfBgkiCwAxjhVopnppwEApxQqhnyQ+VkAAAAASUVORK5CYII=) center center no-repeat;
+            }
+        }
+        button {
+            margin-right: 0;
+            padding: 2px 5px;
+        }
+    }
+    .body-facet {
+        .body-image-container {
+            flex: 0 1 50%;
+            @media screen and (max-width: $screen-xs-min) {
+                flex: 0 1 100%;
+            }
+        }
+        .body-list {
+            flex: 0 1 50%;
+            @media screen and (max-width: $screen-xs-min) {
+                flex: 0 1 100%;
+            }
+        }
+        .body-inset-container {
+            flex: 0 1 60%;
+            text-align: center;
+            .body-inset {
+                img {
+                    width: 55px;
+                    height: 55px;
+                }
+            }
+            .body-inset:after {
+                width: 55px;
+                height: 55px;
+            }
+        }
+        .body-list-narrow {
+            flex: 0 1 40%;
+        }
+    }
+    .body-facet :nth-child(1) {
+        order: 1;
```

Comment:
REQUIRED CHANGE: Based on- https://developer.mozilla.org/en-US/docs/web/css/order, 
is seems you need a second order with webkit prefix. Please see the link.

Reply:
Thank you for catching this!